src/tools/rados/rados.cc

```diff
@@ -2405,11 +2653,17 @@ static int rados_tool_common(const std::map < std::string, std::string > &opts,
     string oid(nargs[1]);
     map<std::string, bufferlist> attrset;
     bufferlist bl;
+
```

Comment:
@chardan i'd suggest use following pattern to avoid repeating.

```c++
#ifdef WITH_LIBRADOSSTRIPER
if (use_striper) {
  ret = striper.getxattrs(oid, attrset);
} else 
#endif
{
  ret = io_ctx.getxattrs(oid, attrset);
}
```

Reply:
I considered that, but it risks introducing subtly different behavior in that the current implementation has both the presence of a striper object /and/ a runtime flag. That said, the flag will always be set to true if creating the striper succeeds.  I'd rather see all the plumbing in here get reworked...